README.md

```diff
@@ -4,6 +4,7 @@ Ballerina GraphQL Library
   [![Build](https://github.com/ballerina-platform/module-ballerina-graphql/actions/workflows/build-timestamped-master.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerina-graphql/actions/workflows/build-timestamped-master.yml)
   [![codecov](https://codecov.io/gh/ballerina-platform/module-ballerina-graphql/branch/master/graph/badge.svg)](https://codecov.io/gh/ballerina-platform/module-ballerina-graphql)
   [![Trivy](https://github.com/ballerina-platform/module-ballerina-graphql/actions/workflows/trivy-scan.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerina-graphql/actions/workflows/trivy-scan.yml)
+  [![GraalVM Check](https://github.com/ballerina-platform/module-ballerina-graphql/actions/workflows/build-with-bal-test-native.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerina-graphql/actions/workflows/build-with-bal-test-native.yml)
```

Comment:
Need to get the badge from `img.shields.io/github` else the label cannot be overwritten as `GraalVM Check`
```suggestion
  [![GraalVM Check](https://img.shields.io/github/workflow/status/ballerina-platform/module-ballerina-graphql/Build%20with%20bal%20test%20native?label=GraalVM%20Check)](https://github.com/ballerina-platform/module-ballerina-graphql/actions/workflows/build-with-bal-test-native.yml)
```